nextstrain_profiles/100k/config.yaml

```diff
@@ -0,0 +1,33 @@
+
+S3_DST_BUCKET: "nextstrain-ncov-private/100k"
+S3_DST_ORIGINS: [needed-for-workflow-but-unused]
+deploy_url: needed_for_workflow_but_unused
+
+custom_rules:
+  - workflow/snakemake_rules/export_for_nextstrain.smk
+
+# Note: unaligned sequences are provided as "aligned" sequences to avoid an initial full-DB alignment
+inputs:
+  - name: gisaid
+    metadata: "s3://nextstrain-ncov-private/metadata.tsv.gz"
+    aligned: "s3://nextstrain-ncov-private/sequences.fasta.xz"
+    skip_sanitize_metadata: true
+
+builds:
+  100k:
+    subsampling_scheme: 100k_scheme
+
+# remove sequences without division label in US
+filter:
+  exclude_where: "division='USA'"
+
+subsampling:
+  100k_scheme:
+    50k_early:
+      group_by: "year month region"
```

Comment:
I suspect that `year month country` will give a better basis for further analysis, as Europe will be dominated by Germany and the UK, Africa will be dominated by South Africa and North America will be dominated by the US.

Reply:
Done :)